kubernetes/argocd/stacks/bmes-wordpress/main.yml

```diff
@@ -0,0 +1,134 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wordpress-pvc
+  namespace: bmes-wordpress
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: nfs-csi
+  resources:
+    requests:
+      storage: 20Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: database-pvc
+  namespace: bmes-wordpress
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: nfs-csi
+  resources:
+    requests:
+      storage: 2Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bmes-website
+  namespace: bmes-wordpress
+spec:
+  strategy:
+    type: Recreate # hell on earth.
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: bmes-website
+  template:
+    metadata:
+      annotations:
+      labels:
+        app.kubernetes.io/name: bmes-website
+    spec:
+      volumes:
+        - name: wp-data
+          persistentVolumeClaim:
+            claimName: wordpress-pvc
+        - name: wp-db
+          persistentVolumeClaim:
+            claimName: database-pvc
+      containers:
+        - name: wordpress-db
+          image: "docker.io/mysql"
+          volumeMounts:
+            - name: wp-db
+              mountPath: /var/lib/mysql
+          env:
+            - MYSQL_ROOT_PASSWORD: notASecret
+            - MYSQL_DATABASE: wordpress
+        - name: wordpress
+          image: "docker.io/wordpress"
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          volumeMounts:
+            - name: wp-data
+              mountPath: /var/www/html
+          env:
+            - WORDPRESS_DB_HOST: localhost
+            - WORDPRESS_DB_USER: root
+            - WORDPRESS_DB_PASSWORD: notASecret
+            - WORDPRESS_DB_NAME: wordpress
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
+          resources:
+            limits:
+              cpu: '1000m'
+              memory: 1024Mi
+            requests:
+              cpu: '100m'
+              memory: 256Mi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: bmes-wordpress
+  name: wordpress
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: bmes-website
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: bmes
+  namespace: bmes-wordpress
+  annotations:
+    external-dns.alpha.kubernetes.io/target: app.acmuic.org
+spec:
+  entryPoints:
+    - websecure
+    - web
+  routes:
+    - kind: Rule
+      match: "Host(`convergehealth.cc`) || Host(`www.convergehealth.cc`)"
+      services:
+        - kind: Service
+          name: bmes # Perhaps move to new file.
```

Comment:
On line 97 above, you name the service `wordpress`, not `bmes` as you define in this IngressRoute.